lib/zendesk_apps_support/package.rb

```diff
@@ -278,6 +279,28 @@ def app_js
       read_file('app.js')
     end
 
+    def location_icons
+      location_icons = {}
+      manifest.locations.each do |host, locations|
+        locations.keys.each do |location|
```

Comment:
Wasn't there a different/better way to get locations now? (maybe not, i can't remember)